### ISSUE TITLE
Issue with UbuntuTouch_Partition_Creator

### ISSUE THREAD

**Issue by slugdude** [jshafer817/UbuntuTouch#1]
The picture is rather self-explanatory:

![image](https://cloud.githubusercontent.com/assets/10201344/11458775/e2a8a4ca-96be-11e5-894a-a51557e8b2a4.png)

**Comment by jshafer817** [comment]
I ran into this myself not to long ago, and I am really not sure what caused it. You seem to have webos installed... not enough info either.. my guess is you may not have had enough freespace to start with?

**Comment by slugdude** [comment]
According to THIS: http://forum.xda-developers.com/showthread.php?t=2426924 a little more than 2GB is needed, and I have at least 5GB unallocated. I'm able to create partitions manually using another tool so for the short term can you tell me what partitions need to be created? (The partitions that UbuntuTouch_Partition_Creator should create).

About the actual error I'd guess that the size has been specified in that command as resizefat -1032M <device> (hence the error about options) but it should really be resizefat <device> 1032M (according to that output),but the code used to create UbuntuTouch_Partition_Creator isn't on here (?) so I can't help more.

**Comment by jshafer817** [comment]
https://raw.githubusercontent.com/jshafer817/UbuntuTouch/master/Source%20to%20compile/device/hp/tenderloin/init.tenderloin.rc

This is what it is doing:
mount ext4 /dev/store/ut-system /system wait ro barrier=1
mount ext4 /dev/store/ut-data /data wait noatime nodev barrier=1 noauto_da_alloc
mount ext4 /dev/store/ut-cache /cache wait noatime nosuid nodev barrier=1

Last time I used.... touchpad toolbox from jcsullins to create the freespace, then rebooted into webos as root and created the new volume groups... 

You could try this:
http://forum.xda-developers.com/showthread.php?t=2426924&page=45#post64091286

login with novaterm as root, and last I tried in webos 3.0.5 it worked...
Should help... At least the lvcreate stuff. =)

**Comment by slugdude** [comment]
I just tried that and it successfully created the partitions, then when I installed the zips and rebooted all I got was a blank screen? I am able to adb into it, and I get presented with this:

 BusyBox v1.20.2 (Ubuntu 1:1.20.0-8.1ubuntu1) built-in shell (ash)
Enter 'help' for a list of built-in commands.

**Comment by jshafer817** [comment]
Did you install the cm10.1 rom to go along with it? That had a 2.6 kernel..

**Comment by slugdude** [comment]
Yes, it's the one that was in that thread. Should I use different ones?

**Comment by slugdude** [comment]
Will using different cm builds work or have you modified the one given to suit Ubuntu touch?
